R/bootstrap-layout.R

```diff
@@ -546,6 +559,8 @@ splitLayout <- function(..., cellWidths = NULL, cellArgs = list()) {
 #'   `fillRow`/`fillCol`, or some other HTML element whose height is
 #'   not determined by the height of its contents.
 #'
+#' @family layout functions
+#'
```

Comment:
I don't think `fillRow()` should be included in the layout family. If you're going to include `fillRow()`, why not include `fluidRow()`?